frontend/components/facebook.js

```diff
@@ -93,12 +100,16 @@ class Facebook {
     } else if (e.event === 'opt_in') {
       macros.log('Opt in was clicked!', e);
 
+      user.downloadUserData(100);
```

Comment:
Maybe we can have some better logic than retry 100 times? I'd have to think about this though.. is a bit yucky

Reply:
🤔 Will sit on this for now, and try to look at the othe rother thing